### PR TITLE
Document new saucectl npm dependencies feature

### PR DESCRIPTION
**Pull request by alexplischke**
### Description

Describe the new feature that's coming in https://github.com/saucelabs/saucectl/pull/560

### CODE REVIEW

**Review by tianfeng92 — APPROVED** [review]

File: docs/web-apps/automated-testing/cypress/yaml.md [hunk 1]
```diff
@@ -338,6 +338,31 @@ Specifies any NPM packages that are required to run tests and should, therefore,
...
+
+Take note that the syntax is different from `packages`. It's a simple **list** of dependencies, without the need to specify the version.
+
+```yaml
+npm:
+  dependencies:
+    - lodash
+```
+
+To use this feature, please also ensure that `node_modules` is not ignored via `.sauceignore`.
```
Comment (lysannep) [inline docs/web-apps/automated-testing/cypress/yaml.md:358]:
```suggestion
To use this feature, make sure that `node_modules` is not ignored via `.sauceignore`.
```

File: docs/web-apps/automated-testing/playwright/yaml.md [hunk 2]
```diff
@@ -330,6 +330,31 @@ Specifies any NPM packages that are required to run tests and should, therefore,
...
+
+Take note that the syntax is different from `packages`. It's a simple **list** of dependencies, without the need to specify the version.
+
+```yaml
+npm:
+  dependencies:
+    - lodash
+```
+
+To use this feature, please also ensure that `node_modules` is not ignored via `.sauceignore`.
```
Comment (lysannep) [inline docs/web-apps/automated-testing/playwright/yaml.md:350]:
```suggestion
To use this feature, make sure that `node_modules` is not ignored via `.sauceignore`.
```

File: docs/web-apps/automated-testing/testcafe/yaml.md [hunk 3]
```diff
@@ -331,6 +331,31 @@ Specifies any NPM packages that are required to run tests and should, therefore,
...
+
+Take note that the syntax is different from `packages`. It's a simple **list** of dependencies, without the need to specify the version.
+
+```yaml
+npm:
+  dependencies:
+    - lodash
+```
+
+To use this feature, please also ensure that `node_modules` is not ignored via `.sauceignore`.
```
Comment (lysannep) [inline docs/web-apps/automated-testing/testcafe/yaml.md:351]:
```suggestion
To use this feature, make sure that `node_modules` is not ignored via `.sauceignore`.
```

**Review by lysannep — APPROVED** [review]
Looks good ✨ I added a suggestion to remove "please". Thank you!

**Pull request merged**